WTWSMS/common/religions/00_religions.txt

```diff
@@ -4935,28 +5001,49 @@ jewish_group = {
 		}
 		
 		unit_modifier = {
-			levy_size = 0.3
+			levy_size = 0.35
 			
 			retinuesize_perc = 0.5
 			
 			key = "OFF_RELIGION"
 		}
+
+		unit_home_modifier = {
+			land_morale = 0.66
+			light_infantry_defensive = 0.3
+			heavy_infantry_defensive = 0.4
+			knights_defensive = 0.3
+			archers_defensive = 0.2
+			garrison_size = 0.25
+			winter_supply = 0.05 # Mountainous nature
+
+			key = "DEF_RELIGION_HOME_TERRITORY"
+		}
 		
 		character_modifier = {
-			martial  = 2
+			martial  = 4
+			combat_rating = 1
+			commander_limit = 1
+			ai_ambition = 1
+			ai_zeal = 2

```

Comment:
A bonus of positive one is not too much. AI_AMBITION and AI_ZEAL are not shown, nor do they effect the player.